versions/3.0.md

```diff
@@ -2689,14 +2809,22 @@ example:
   "required": [
     "name"
   ],
-  "examples": [
-  {
-    "name": "Puma",
-    "id": 1
-  }, {
-    "name": "Ferguson",
-    "id": 2
-  }]
+  "examples": {
```

Comment:
Why would we support `example` and `examples` elsewhere but not here?  Was removing this intentional?  It doesn't seem to be because on line 2812 you use `examples`.